src/core/packages/chrome/layout/core-chrome-layout/layouts/grid/hack_use_sync_push_flyout.tsx

```diff
@@ -52,14 +53,28 @@ export function useHackSyncPushFlyout() {
           const paddingInline = parsedStyle.find(
             (style) => style.property === 'padding-inline'
           )?.value;
+          let paddingInlineStart = parsedStyle.find(
+            (style) => style.property === 'padding-inline-start'
+          )?.value;
           let paddingInlineEnd = parsedStyle.find(
             (style) => style.property === 'padding-inline-end'
           )?.value;
 
-          const [, end] = paddingInline?.split(' ') ?? ['', ''];
+          const [start, end] = paddingInline?.split(' ') ?? ['', ''];
 
+          paddingInlineStart = paddingInlineStart ?? start;
           paddingInlineEnd = paddingInlineEnd ?? end;
 
+          if (paddingInlineStart) {
```

Comment:
This is never going to be falsey, right?  Because of line 63?  Or am I missing something?

Reply:
it can be falsey if there is no `padding-inline-end` or `padding-inline` styles set on body